### ISSUE TITLE
Add term thickened cortex of the ulna

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10453]
**Preferred term label:**
Thickened cortex of the ulna

**Synonyms**
Broad cortex of ulna- Cortical thickening of the ulna- Thickened cortices of ulna

**Definition (free text, please give PubMed ID)**
Abnormal thickening of the cortex of the ulna. PMID24389367

**Parent term (use hpo.jax.org/app)**
Thickened cortex of long bones HP:0000935
Cortical thickening of the forearm bones HP:0003957
Abnormal morphology of ulna HP:0040071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Caffey Disease (omim114000), for example: PMID24390061

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000812
- label: Thickened cortex of the ulna
- parents: [Thickened cortex of long bones (HP:0000935)](https://hpo.jax.org/app/browse/term/HP:0000935) | [Cortical thickening of the forearm bones (HP:0003957)](https://hpo.jax.org/app/browse/term/HP:0003957)
- pmid: PMID:24390061
- def: Abnormally increased thickness of the cortical layer of the ulna.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10453
- synonyms:
